lib/rubocop/cop/rspec/expect_actual.rb

```diff
@@ -65,6 +76,11 @@ def complex_literal?(node)
           COMPLEX_LITERALS.include?(node.type) &&
             node.each_child_node.all?(&method(:literal?))
         end
+
+        def swap_order(corrector, lhs_arg, rhs_arg)
```

Comment:
In RSpec terminology `lhs_arg` is "expectation target", and `rhs_arg` is "matcher argument", but I agree that at this level we operate nodes.